web/src/engine/osk/build.sh

```diff
@@ -36,7 +36,12 @@ builder_parse "$@"
 
 #### Build action definitions ####
 
-builder_run_action configure verify_npm_setup
+if builder_start_action configure; then
```

Comment:
builder_describe_outputs (line 32) needs `configure` to point to the font, because `/node_modules` is already present, so this step is skipped.